mat64/dense.go

```diff
@@ -379,7 +379,9 @@ func (m *Dense) Clone(a Matrix) {
 
 // Copy makes a copy of elements of a into the receiver. It is similar to the
 // built-in copy; it copies as much as the overlap between the two matrices and
-// returns the number of rows and columns it copied.
+// returns the number of rows and columns it copied. If a aliases the receiver
+// and is a transposed Dense or Vector, with a non-unitary increment, Copy will
+// panic.
```

Comment:
Maybe we should explicity add a CopyRow then? If I wanted to copy a row from one part of a matrix to another, I would use a view. This does not require using a transpose. 
````
    r := a.View(r, 0, 1, cols)
    a.View(dst,0,1,cols).(*mat.Dense).Copy(r)
````
If I explicitly know it's a row, then I could call CopyRow (which can be added). There's overlap with SetRow, but they're slightly different.